### PR TITLE
1658: main table design

### PR DESCRIPTION
**Pull request by sutt0n**
Note: Ellipsis only appears for vertically-oriented message (limited by 3 lines of text) on Chrome.

### CODE REVIEW

File: web-client/src/styles/tables.scss [hunk 1]
```diff
@@ -64,29 +64,53 @@ table.dashboard {
...
-      margin: 0;
+    &.message-select-control {
+      vertical-align: middle;
     }
-  }
 
-  td.section-queue-row {
-    padding-top: 14px;
-  }
+    .message-queue-row {
```
Comment (kkoskelin) [inline web-client/src/styles/tables.scss]: Did you mean `&.message-queue-row`?

File: web-client/src/styles/tables.scss [hunk 2]
```diff
@@ -64,29 +64,53 @@ table.dashboard {
...
-    width: 44px;
+    div.message-document-title {
+      overflow: hidden;
+      max-width: units('tablet');
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    div.message-document-detail {
+      display: -webkit-box;
```
Comment (kkoskelin) [inline web-client/src/styles/tables.scss:102]: I'm unsure about using the `-webkit` prefixed properties and values.  Is this our only option? cc: @ericsorenson

File: web-client/src/styles/tables.scss [hunk 3]
```diff
@@ -64,29 +64,53 @@ table.dashboard {
...
+      max-height: 57px; /* fallback */
+      /* autoprefixer: ignore next */
+      -webkit-box-orient: vertical;
+      font-size: 15px;
+      -webkit-line-clamp: 3; /* number of lines to show */
+      line-height: 19px; /* fallback */
+      text-overflow: ellipsis;
+    }
+
+    .has-icon {
```
Comment (kkoskelin) [inline web-client/src/styles/tables.scss]: `&.has-icon`?

File: web-client/src/styles/tables.scss (hunk 2 above)
Comment (matthopson) [reply web-client/src/styles/tables.scss:102]: @kkoskelin This is a compromise. As we discussed this morning in our meeting, this is pretty much the only option for getting this to wrap / truncate as expected - and it only works in webkit browsers. Luckily our internal users will be using that. So it's either have it for our target case, or not at all.

I would definitely be interested in further discussion around this though, and remove later if necessary.

Comment (kkoskelin) [reply web-client/src/styles/tables.scss:102]: ok, yeah, I definitely didn't fully _wrap_ my head around the conversation at the time. 🥁

Comment (matthopson) [reply web-client/src/styles/tables.scss:102]: hahaha, well done, sir.

**Review by ericsorenson — APPROVED** [review]

**Review by codyseibert — APPROVED** [review]

**Pull request merged**